### PR TITLE
Added AkshayHere

### PR DESCRIPTION
**Pull request by AkshayHere**

### CODE REVIEW

File: Data/member.json
Comment (NS007-dev) [inline Data/member.json]: Please remove the comma after the brace on line 496 @AkshayHere

**Review by NS007-dev — CHANGES_REQUESTED** [review]

File: Data/member.json
Comment (NS007-dev) [inline Data/member.json]: @AkshayHere Please can you also align the braces, so they are formatted?

**Review by NS007-dev — CHANGES_REQUESTED** [review]

**Review by NS007-dev — APPROVED** [review]
Thank you for fixing this. Just fix the review I placed above, and it should be fine @AkshayHere

**Comment by AkshayHere** [comment]
Formatted the JSON file to maintain coding standards

**Review by NS007-dev — APPROVED** [review]
~~I am not sure how all of it got deleted, but it seems as though you have restored them all.~~
Thank you for fixing it! @AkshayHere 
We will have to wait for a maintainer to merge this.

**Comment by NS007-dev** [comment]
> 
> 
> Formatted the JSON file to maintain coding standards


Thank you for formatting the whole file! I just realised that! Thank you for that! I forgot to say that it was only for line 490, but thank you for doing the whole file @AkshayHere

**Comment by AkshayHere** [comment]
> 
> 
> > Formatted the JSON file to maintain coding standards
> 
> Thank you for formatting the whole file! I just realised that! Thank you for that! I forgot to say that it was only for line 490, but thank you for doing the whole file @AkshayHere

Sure. I would recommend adding some ESLint features to standardize the code across the repo.

**Comment by NS007-dev** [comment]
> 
> 
> > > Formatted the JSON file to maintain coding standards
> > 
> > 
> > Thank you for formatting the whole file! I just realised that! Thank you for that! I forgot to say that it was only for line 490, but thank you for doing the whole file @AkshayHere
> 
> Sure. I would recommend adding some ESLint features to standardize the code across the repo.

Yes, it has been said in this issue #134

**Comment by AkshayHere** [comment]
> 
> 
> > > > Formatted the JSON file to maintain coding standards
> > > 
> > > 
> > > Thank you for formatting the whole file! I just realised that! Thank you for that! I forgot to say that it was only for line 490, but thank you for doing the whole file @AkshayHere
> > 
> > 
> > Sure. I would recommend adding some ESLint features to standardize the code across the repo.
> 
> Yes, it has been said in this issue #134

Great :). Thanks for the same.

**Review by MrSlimCoder — APPROVED** [review]
Looks Good

**Comment by MrSlimCoder** [comment]
@AkshayHere you need to work on conflicts and formatting then I will merge it.

**Pull request closed without merging**
